src/Uno.UI/Helpers/WinUI/ResourceAccessor.ResourceKeys.cs

```diff
@@ -132,7 +132,11 @@ internal partial class ResourceAccessor
 		public const string SR_TabViewScrollIncreaseButtonTooltip = "TabViewScrollIncreaseButtonTooltip";
 		public const string SR_NumberBoxUpSpinButtonName = "NumberBoxUpSpinButtonName";
 		public const string SR_NumberBoxDownSpinButtonName = "NumberBoxDownSpinButtonName";
+		public const string SR_DragViewLinkCaption = "DragViewLinkCaption";
+		public const string SR_DragViewCopyCaption = "DragViewCopyCaption";
+		public const string SR_DragViewMoveCaption = "DragViewMoveCaption";
```

Comment:
This entire file is copy-pasted from the WInUI Library. Adding special keys like this could easily get lost if every the file is copied over again. I don't think custom keys should go here.

Reply:
RIght, did not take that into consideration. Good catch.